.github/workflows/integration_test_charm.yaml

```diff
@@ -412,21 +412,41 @@ jobs:
       - name: juju status
         timeout-minutes: 1
         if: ${{ success() || (failure() && steps.tests.outcome == 'failure') }}
-        run: juju status --color --relations | tee ~/logs/juju-status.txt
+        run: |
+          MODELS=$(juju models --format json | jq -r '.models[] | select(."short-name" != "controller") | ."short-name"' | sed ':a;N;$!ba;s/\n/ /g')
+          for MODEL in $MODELS
+          do
+            juju status --model="$MODEL" --color --relations | tee "$HOME/logs/juju-status-$MODEL.txt"
+            printf "\n"
+          done
       - name: juju debug-log
         timeout-minutes: 3
         if: ${{ success() || (failure() && steps.tests.outcome == 'failure') }}
         # TODO future improvement: show models/ logs in juju-debug-log.txt to match `juju debug-log`
         run: |
+          MODELS=$(juju models --format json | jq -r '.models[] | select(."short-name" != "controller") | ."short-name"' | sed ':a;N;$!ba;s/\n/ /g')
```

Comment:
```suggestion
          declare -a MODELS=($(juju models --format json | jq -r '.models[] | select(."short-name" != "controller") | ."short-name"'))
```

Reply:
Thanks for the improvement, Alex! Fixed in the same commit commented on https://github.com/canonical/data-platform-workflows/pull/195#discussion_r1646513966.